GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1302,7 +1353,7 @@ protected void createRawDataHeader(Workbook wb,
                     if (qgSheetMap.containsKey(group.getKeyId()))   {
                         Sheet groupSheet = qgSheetMap.get(group.getKeyId());
                         int metaEnd = addMetaDataHeaders(groupSheet, true);
-                        addComment(groupSheet);
+//                        addComment(groupSheet, 0, 0, REPORT_COMMENT);
```

Comment:
if its commented out I think lets remove it completely for now.  When we need it again we will add it in.